app/controllers/traces_controller.rb

```diff
@@ -262,6 +262,25 @@ def icon
 
   private
 
+  def deny_html_access_for_current_user(exception)
+    user_block = current_user.blocks.active.take
+    if exception.action == :new || exception.action == :create
```

Comment:
You can use `action_name` to get the current action, in any controller. This might be easier than passing the exception around to get the same information.

Reply:
ok, although it's a string instead of a symbol

`deny_access` also receives and ignores an exception